src/modules/contact/contact.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { ContactService } from './contact.service';
+import { Contacts } from './dto/contact.dto';
+
+@Controller('contacts')
+export class ContactController {
+  constructor(private readonly contactService: ContactService) {}
+
+  @Get()
+  async getAllContacts(@Res() response) {
+    const data = await this.contactService.getAllContacts();
+    return response.status(200).send(data);
```

Comment:
nao precisa forcar esse status 200. Apenas assim:
```suggestion
    return data
```
Deixa que o nestjs trata isso por baixo

Reply:
Ok